### PR TITLE
New package: quiterss-0.18.12

### PR DESCRIPTION
**Pull request by nocmd**

### CODE REVIEW

File: srcpkgs/quiterss/template [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'quiterss'
+pkgname=quiterss
+version=0.18.12
+revision=1
+create_wrksrc=yes
+only_for_archs="i686 x86_64"
```
Comment (maxice8) [inline srcpkgs/quiterss/template]: Please remove this

File: srcpkgs/quiterss/template [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+build_style=qmake
+configure_args="CONFIG+=release PREFIX=/usr QuiteRSS.pro"
```
Comment (maxice8) [inline srcpkgs/quiterss/template]: PREFIX=/usr is already set by qmake

**Comment by Chocimier** [comment]
all *-devel should be placed in makedepends, not in hostmakedepends

sqlite, qt5-multimedia, qt5-webkit are not necessary in depends, they are found automagically

short_desc usually do not contain name of program

**Comment by maxice8** [comment]
Please squash all commits into one

**Comment by maxice8** [comment]
Please squash all commits into one

**Pull request closed without merging**